cwarhm/data_specific_processing/era5.py

```diff
@@ -0,0 +1,227 @@
+
+import netCDF4 as nc4
+from pathlib import Path
+import time, os
+
+
+def merge_era5_forcing(forcingPath, mergePath, years_str):
+    """ Combine separate surface and pressure level downloads
+    Creates a single monthly `.nc` file with SUMMA-ready variables for further processing. # Combines ERA5's `u` and
+    `v` wind components into a single directionless wind vector.
+
+    :param forcingPath: path to raw ERA5 surface and pressure level data
```

Comment:
with sphinx it is possible to link to another function in the code:
https://www.sphinx-doc.org/en/master/tutorial/describing-code.html#cross-referencing-python-objects
this might be relevant later, as this function follows a specific download function.

so this line could be 
:param forcingPath: path to folder with raw ERA5 surface and pressure level data, as downloaded with :py:func:[Insert downloadfunctions]

just adding this as a reminder for later

Reply:
Agree